src/Temperature.hpp

```diff
@@ -97,13 +97,13 @@ class Temperature
 
 };
 
-static Temperature operator+( Temperature a, Temperature b );
+Temperature operator+( Temperature a, Temperature b );
 
-static Temperature operator-( Temperature a, Temperature b );
+Temperature operator-( Temperature a, Temperature b );
 
-static Temperature operator*( Temperature a, double b );
+Temperature operator*( Temperature a, double b );
 
-static Temperature operator*( double a, Temperature b );
+Temperature operator*( double a, Temperature b );
 
 static std::ostream& operator << (std::ostream& os, Temperature temperature);
```

Comment:
I guess this `static` must be removed as well (this gives a `[-Wunused-function]` warning, too).